modules/cms/controllers/ThemeOptions.php

```diff
@@ -143,4 +151,22 @@ protected function findThemeObject($name = null)
 
         return $theme;
     }
+
+    /**
+     * Register language namespace if available for the theme
+     *
+     * @param string $dirName
+     *
+     * @throws ApplicationException
+     */
+    protected function registerLocalization(string $dirName)
+    {
+        /** @var Theme $theme */
```

Comment:
```suggestion
```
This doesn't add anything useful.

Reply:
It does for IDE (autocompletion will be enabled) and for devs who won't need to guess which object is there.